Documentation/In-depth/SystemRequirements/Index.rst

```diff
@@ -133,3 +133,9 @@ are available.
   * postgresql (if you use PostgreSQL as DBMS)
   * sqlsrv (if you use SQL Server as DBMS)
   * sqlite (if you use SQLite as DBMS)
+
+..warning::
+
+  * Mac-Users: On macOS 10.13.6 and probably also 10.14.* (unconfirmed) pcre causes troubles, with brew as well as native or MAMP installations. It is recommended to use php 7.2. More info can be found here:
```

Comment:
The problem only exists in 7.3? Might want to point that out. (I can push another change for that).

Also, I did not realize this is a not-TYPO3-specific thing. We usually document TYPO3 things only in docs*, but I do agree that is not done religiously everywhere and as already mentioned in Slack, here the benefit would probably outweigh the concerns. 

`*` reason for restricting docs to TYPO3 specific things: docs would bloat up indefinitely and it is already a lot of work to maintain everything. Also, non TYPO3 specific things are often already documented well elsewhere.